core/templates/components/rubrics-editor/rubrics-editor.component.html

```diff
@@ -65,18 +67,30 @@
             </button>
             <button type="button"
                     class="btn btn-success protractor-test-save-rubric-explanation-button"
-                    [disabled]="!this.isExplanationValid(this.rubric.getDifficulty(), idx)"
+                    [disabled]="!this.isExplanationValid(this.rubric.getDifficulty(), idx)
+                      || !this.isExplanationLengthValid(this.editableExplanations[this.rubric.getDifficulty()][idx])"
                     (click)="this.saveExplanation(this.rubric.getDifficulty(), idx)">
               Save
             </button>
           </div>
+          <br>
+          <div *ngIf="!this.isMediumLevelExplanationValid(this.rubric.getDifficulty(), this.editableExplanations[this.rubric.getDifficulty()][idx])"
+               class="rubric-error-message">
+            There should be at least one explanation for the Medium level rubric.
+          </div>
+          <div *ngIf="!this.isExplanationLengthValid(this.editableExplanations[this.rubric.getDifficulty()][idx])" class="rubric-error-message">
+            Explanation length should be below 300 characters.
+          </div>
+          <div class="oppia-card-content-limit-error" *ngIf="this.isTotalExplanationsLengthReached()">
+            There can be total of 10 explanations considering all levels.
```

Comment:
@EricZLou So here I have created a div tag to show the warning and it will be shown while the user is creating the 10th explanation and after that `+ ADD EXPLANATION FOR DIFFICULTY` button will be disabled but what I want is to hover over the button when it's disabled and show the warning. I wasn't able to quite perform it.
Thanks

Reply:
Maybe try using `ng-disabled` and add a `title` attribute to the `+ ADD EXPLANATION FOR DIFFICULTY` button? Like:
```
<button title="some explanation here">
```

Reply:
Not too sure if that will work though. I will keep looking. I don't know off the top of my head.

Reply:
Sure, I'll try it out.
Thanks

Reply:
No, this approach is not working here. I checked it out.

Reply:
So, I found this material on angular where we can use tooltip. Please do let me know if it's okay to use in our codebase.
https://material.angular.io/components/tooltip/api
Thanks

Reply:
@EricZLou

Reply:
Will get back to you on this! Thanks for the ping.

Reply:
Let's just keep what you have now, and skip the tooltip. We can add the tooltip in a future PR!